provider/workspace_preset.go

```diff
@@ -119,9 +135,82 @@ func workspacePresetDataSource() *schema.Resource {
 								},
 							},
 						},
+						"autoscaling": {
+							Type:        schema.TypeList,
+							Description: "Configuration block that defines autoscaling behavior for prebuilds. Use this to automatically adjust the number of prebuild instances based on a schedule.",
+							Optional:    true,
+							MaxItems:    1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"timezone": {
+										Type:        schema.TypeString,
+										Description: "The timezone to use for the autoscaling schedule (e.g., \"UTC\", \"America/New_York\").",
```

Comment:
Nit: can you include a link to an acceptable list of TZs here? I _think_ https://en.wikipedia.org/wiki/List_of_tz_database_time_zones suffices but I'm not sure.

https://pkg.go.dev/time#LoadLocation says it references the "IANA Time Zone database".

Reply:
See the help output of `coder schedule {start|stop}` for some sample wording. The phrase "timezone" is slightly misleading here as it's the _location_ that influences the timezone depending on wibbly wobbly timey wimey stuff.